test/cf/constraints/all_namespace_must_have_cost_center.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRequiredLabels
+metadata:
+  name: pod-must-have-gk
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Namespace"]
+#    namespaceSelector:
```

Comment:
is it important to have this commented block? not sure what it's used for...

Reply:
whoops, forgot to remove that (it used to be a constraint for Pods).